website/dbt-versions.js

```diff
@@ -0,0 +1,31 @@
+exports.versions = [
+  {
+    version: "1.2",
```

Comment:
```suggestion
    version: "1.1",
```

Reply:
@JKarlavige Should I comment this out until the 1.1 release?

Reply:
@runleonarun committed your suggestion and commented out 1.1!